### PR TITLE
Theme

### PR DESCRIPTION
**Pull request by vijaytembugade**
Added Dark theme 
Added its context and Effect!

### CODE REVIEW

File: src/Components/NoteEditor/NoteEditor.js [hunk 1]
```diff
@@ -24,7 +24,12 @@ const NoteEditor = () => {
           </div>
           <div className="flex-container">
             <button className="btn btn-primary-outline">Save</button>
-            <button className="btn  btn-primary-outline">Cancle</button>
+            <button
+              className="btn  btn-primary-outline"
+              onClick={() => setShowNoteEditor(false)}
+            >
+              Cancle
```
Comment (horsemaker) [inline src/Components/NoteEditor/NoteEditor.js:31]: typo

File: src/Contexts/themeContext/theme-context.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import { createContext, useState, useContext, useLayoutEffect } from "react";
+
+const ThemeContext = createContext();
+
+const ThemeProvider = ({ children }) => {
+  const prevTheme = localStorage.getItem("theme") || "";
+  const [theme, setTheme] = useState(prevTheme);
+
+  useLayoutEffect(() => {
```
Comment (horsemaker) [inline src/Contexts/themeContext/theme-context.js:9]: Nice use of `useLayoutEffect` 👌

**Review by horsemaker — APPROVED** [review]
Looks good to go! 👍

File: src/Components/AllNotes/AllNotes.js [hunk 3]
```diff
@@ -19,43 +19,7 @@ const AllNotes = () => {
...
-              >
-                Cancle
-              </button>
-            </div>
-          </div>
-        </div>
-      )}
+      {showNoteEditor && <NoteEditor setShowNoteEditor={setShowNoteEditor} />}
 
       <div dangerouslySetInnerHTML={{ __html: value }}></div>
```
Comment (narharikale) [inline src/Components/AllNotes/AllNotes.js:24]: learned new thing today 🤞

**Review by narharikale — APPROVED** [review]
ui looks clean 🤩

**Pull request merged**